src/handlers/channel.ts

```diff
@@ -422,6 +424,55 @@ export async function editChannel(
   );
 }
 
+export async function editChannelOverwrite(
+  channelID: string,
+  overwrite: Overwrite,
+  reason?: string,
+) {
+  const hasManageChannelPerm = await botHasChannelPermissions(
+    channelID,
+    ["MANAGE_CHANNELS"],
+  );
+  if (!hasManageChannelPerm) {
+    throw new Error(Errors.MISSING_MANAGE_CHANNELS);
+  }
+
+  const { allow, deny, ...info } = overwrite;
+  let channel = cache.channels.get(channelID);
+
+  if (!channel) return Errors.CHANNEL_NOT_FOUND;
+
+  const payload = {
+    permission_overwrites: [
+      ...(channel?.permissionOverwrites || []).map((ro) => ({
```

Comment:
`ro` is not-so explanatory, maybe rename it to `rawOverwrite`?

Reply:
Done